stackrox/base/secret/secret.yaml

```diff
@@ -0,0 +1,722 @@
+---
```

Comment:
@traytorous I'll work on getting secrets into vault and setting up external secrets

Reply:
Resolved